src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryFindCommand.java

```diff
@@ -20,9 +25,12 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all deliveries whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "PREFIX only limited to name, phone, address and order\n"
-            + "Parameters: " + "PREFIX " + "KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " n/Sally";
+            + "Parameters: "
+            + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_PHONE + "PHONE] "
+            + "[" + PREFIX_ADDRESS + "ADDRESS] "
+            + "[" + PREFIX_ORDER + "ORDER]\n"
+            + "Example: " + COMMAND_WORD + " n/Sally p/84738594";
```

Comment:
I believe the UG has changed now to use the `|` format instead

Reply:
Alright, will make changes. Since edit error message is also in this format, i will make it likewise with the `|`